webapp/lib/model.g.dart

```diff
@@ -250,6 +250,7 @@ class MessageStatus {
         if (value.name == valueName) return value;
       }
     }
+    log.warning('unknown MessageStatus $text');
```

Comment:
This is generated, but Luke indicated that we should remove this because it was generating too much noise in the logs. Please comment this out and add a comment above it along the lines of 
`// This is commented out because it generates too much noise in the logs`

I've just been manually removing this line each time I regenerate the model.
I should have commented out the line and added an explanation. Sorry about that.

Reply:
Ah, yes, I think I remember seeing a discussion along these lines, I should have dug it up before just adding the line in. Commented out now with a comment as you suggested. thanks!